src/common/perf_counters.cc

```diff
@@ -190,7 +190,14 @@ void PerfCounters::dec(int idx, uint64_t amt)
   ceph_assert(!(data.type & PERFCOUNTER_LONGRUNAVG));
   if (!(data.type & PERFCOUNTER_U64))
     return;
-  data.u64 -= amt;
+  if (amt > data.u64) { // would wrap
+    assert(false); // Assert in debug builds
+    lderr(m_cct) << "WARNING: PerfCounters::dec() " << data.u64 << " - "
+            << amt << " would wrap. Setting to 0" << dendl;
+    data.u64 = 0;
+  }
+  else
```

Comment:
nit: should have braces around the else clause since there are braces around the if clause

Reply:
ack